prac_05/states_dictionary.py

```diff
@@ -0,0 +1,33 @@
+"""
+CP1404/CP5632 Practical
+State names in a dictionary
+File needs reformatting
+"""
+
+# # TODO: Reformat this file so the dictionary code follows PEP 8 convention
+# STATE_NAMES = {"QLD": "Queensland", "NSW": "New South Wales", "NT": "Northern Territory", "WA": "Western Australia",
```

Comment:
Don't leave the next work in the previous work with in and leave the previous work commented out